Dependencies/xr/Source/ARCore/XR.cpp

```diff
@@ -49,6 +50,34 @@ namespace xr
 
     namespace
     {
+        void log(const char* message)
+        {
+            __android_log_print(ANDROID_LOG_VERBOSE, __FILE__, "%s", message);
+        }
+
+        template<typename TTimeUnit>
```

Comment:
This was useful for doing some quick and dirty "perf testing" as I went. It doesn't really belong here, but I also think it's probably worth keeping around for dev purposes...

Reply:
Probably put this in macro or put in a separate file somewhere else, so that it's clearly a debug thing?